data/json/scenarios/scenarios.json

```
@@ -175,6 +175,16 @@
     ],
     "flags": [ "CHALLENGE", "CITY_START", "LONE_START" ]
   },
+  {
+    "type": "scenario",
+    "ident": "lab_normal",
+    "name": "Locked Lab",
+    "points": -4,
+    "description": "Thanks to your connections you were invited to visit a top-secret laboratory. Unfortunately your visit coincided with the cataclysm and now the lab has gone into lockdown. Find a way out or perish.",
```

I believe `cataclysm` should be capitalized. Needs a comma after "unfortunately" as well

```suggestion
    "description": "Thanks to your connections you were invited to visit a top-secret laboratory. Unfortunately, your visit coincided with the Cataclysm and now the lab has gone into lockdown. Find a way out or perish.",
```

---

and two whitespaces between sentences